src/main/java/cms/gov/madie/measure/resources/MeasureController.java

```diff
@@ -70,12 +71,17 @@
     return ResponseEntity.status(HttpStatus.CREATED).body(savedMeasure);
   }
 
-  @PutMapping("/measure")
+  @PutMapping("/measure/{id}")
```

Comment:
time to update this endpoint to `...measures/ ` or is it a lot of work?

Reply:
crap...I didn't think about that haha it's same workset I just did - updating the URI here, updating the tests, and updating the frontend call as well, but I can do that

Reply:
this is updated now for both backend and frontend